homeassistant/components/wallbox/number.py

```diff
@@ -87,20 +98,36 @@ def __init__(
     @property
     def native_max_value(self) -> float:
         """Return the maximum available current."""
-        return cast(float, self._coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY])
+        return cast(
+            float,
+            (
+                self._coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY]
```

Comment:
Instead of this code, please something like https://github.com/home-assistant/core/blob/dev/homeassistant/components/unifi/switch.py#L178

The advantage is that you don't need to modify the `WallboxNumber`, when you add another number entity

Reply:
I think I changed my approach to something resembling the unifi component. Please let me know if this is what you had in mind.